core/benchmarks/bm-nosonar_node_api.cpp

```diff
@@ -55,25 +50,20 @@ static_assert(gr::traits::block::can_processOne_simd<multiply<float>>);
 #endif
 
 template<typename T, char op>
-class math_bulk_op : public gr::Block<math_bulk_op<T, op>, gr::PortInNamed<T, "in", gr::RequiredSamples<1, N_MAX>>, gr::PortOutNamed<T, "out", gr::RequiredSamples<1, N_MAX>>> {
-    T _factor = static_cast<T>(1.0f);
-
-public:
-    math_bulk_op() = delete;
-
-    explicit math_bulk_op(T factor, std::string name_ = gr::this_source_location()) : _factor(factor) { this->name = name_; }
+struct math_bulk_op : public gr::Block<math_bulk_op<T, op>, gr::PortInNamed<T, "in", gr::RequiredSamples<1, N_MAX>>, gr::PortOutNamed<T, "out", gr::RequiredSamples<1, N_MAX>>> {
```

Comment:
Same as previous comment:Is this block still needed? Can you use newly implemented Math block?

Reply:
The _bulk function is used for the benchmark only and only there to motivate that using SIMD is the better choice w.r.t. performance compared to the other options outlined in the `processBulk(...)` method. We could codify this just for the math operations just for posteriori to document why the `processOne(...)` is equivalent and/or faster , but in general there is no additional value to be kept in the main block lib.